test/bulleted_list_4_space_indent_no_start.json

```diff
@@ -0,0 +1,7 @@
+{
+    "default": true,
+    "MD007": {
+        "start_indented": false,
+        "indent": 4
```

Comment:
I don't mind the extra test case, but this behavior should be covered by existing tests, right?